src/plugins/administrative_templates/comments/commentsmodel.cpp

```diff
@@ -160,11 +160,10 @@ void savePolicies(const QString &pluginName, const QString &fileName, std::share
     delete format;
 }
 
-QString constructCMTLFileName(const QFileInfo &fileName)
+QString constructCMTLFileName(const QFileInfo &fileName, const QString &localeName)
 {
-    QString admlFileName = fileName.filePath();
-    admlFileName.replace(admlFileName.length() - 4, 4, "cmtl");
-
+    QString localFolder = localeName.toLower(); // en-US -> en-us. ru-RU -> ru-ru and etc.
```

Comment:
Why do we changing locale name here? On Linux type OS paths are case sensitive and on smb shares they are not. We support both file and smb variations of the paths so using locale as it is in this case is intentional.